### PR TITLE
Added animations, hovers and focus effects

### PR DESCRIPTION
**Pull request by marsian83**
## Details
- **This PR is related to**: Issue #10 
- **Does it close the issue?**: Yes

**Summary**:
      This PR adds:
      - hover and focus effect on call-to-action buttons in home page, submit buttons and cards
      - slight hover animation to the submit button
      - cards now grow in size when hovered upon and are clickable and would redirect to the same location as the "Learn More" buttons 

**Screenshots**:
Screenshot (26)
Screenshot (27)
Screenshot (28)

### CODE REVIEW

File: home/style.css [hunk 1]
```diff
@@ -91,6 +91,66 @@
...
+  }
+
+  .card .call-to-action:focus,.old-card .call-to-action:focus{
+    background-color: #f96400;
+    box-shadow: 0px 0px 15px rgba(0,0,0,0.69);
+  }
+
+  .card:hover,.old-card:hover{
+    transform: scale(1.038);
+  }
```
Comment (Akshu-on-github) [inline home/style.css]: Please move this block of code to the `event-cards.css` file

Additionally, when there are two or more classes sharing the same CSS, please follow the following convention:
```css
.class1,
.class2 {
/* code */
}
```

**Review by Akshu-on-github — CHANGES_REQUESTED** [review]
Hey there!
The effects all look good to me - just need a few changes for consistency with the codebase and we're good to merge it

**Comment by marsian83** [comment]
changed have been made as per the request

**Review by Akshu-on-github — APPROVED** [review]
LGTM ✅

**Pull request merged**